### ISSUE TITLE
packages/required import in design-system/dist/scss/settings/uswds/_index.scss not found

### ISSUE THREAD

**Issue by francesruiz** [CMSgov/design-system#842]
**Describe the bug**
An import of packages/required on line 28 of design-system/dist/scss/settings/uswds/_index.scss is not found.

**To Reproduce**

I am using the npm package, with scss and importing ```@import '@cmsgov/design-system/dist/scss/index';``` in my projects main scss file.

We were previously on 2.0.0 which works as expected.

Today I tried updating to version 2.1.0. That resulted in my build failing due to the following webpack error:

```
@import packages/required
 File to import not found or unreadable: packages/required.
my-project-directory/node_modules/@cmsgov/design-system/dist/scss/settings/uswds/_index.scss (line 28, column 1)
```

**Expected behavior**
This error would not appear.

**Desktop (please complete the following information):**
macOS Catalina 10.5.7 node version 14.13.0

**Smartphone (please complete the following information):**
Not applicable

**Additional context**
It appears that all dependencies (including uswds) installed as expected with our npm install. So we do not believe there is anything incorrect about our directory structure within node_modules that would be causing this issue.

**Comment by nichia** [comment]
Thanks for submitting this @francesruiz.
There seems to be an issue with the `@import packages/required` when importing the scss index in a project's main scss file. This issue is tracked [here](https://jira.cms.gov/browse/WNMGDS-656)

**Comment by bernardwang** [comment]
This has been fixed in the 2.1.1 hotfix release
